pkgs/development/tools/misc/gdb/default.nix

```diff
@@ -48,6 +48,12 @@ stdenv.mkDerivation rec {
 
   patches = [
     ./debug-info-from-env.patch
+
+    # Backport musl fix
+    (fetchpatch {
+      url = "https://sourceware.org/git/?p=binutils-gdb.git;a=patch;h=2e977d9901393ea1bacbe1896af0929e968bc811";
```

Comment:
That seems a fix but does this actually related to musl?

Subject: [PATCH] gdbserver/linux-low.cc: Fix a typo in ternary operator

Reply:
Yes.  The new musl removes the LFS64 functions, e.g. pread64.  gdb has a path for if those functions don't exist, but it had a typo that caused it not to compile, which this patch fixes.